src/trace_senders/http.js

```diff
@@ -19,7 +19,10 @@ const session = axios.create({
 
 module.exports.sendTrace = function sendTrace(traceObject) {
     utils.debugLog(`Posting trace to ${config.getConfig().traceCollectorURL}`);
-    utils.debugLog(`trace: ${JSON.stringify(traceObject, null, 2)}`);
+
+    if ((process.env.EPSAGON_DEBUG || '').toUpperCase() === 'TRUE') {
```

Comment:
why did you add the condition?
is the condition is not part of debugLog?

Reply:
to avoid stringify when not in debug mode